code/__DEFINES/research/anomalies.dm

```diff
@@ -9,10 +9,13 @@
 #define ANOMALY_CORE_VORTEX /obj/item/assembly/signaler/anomaly/vortex
 /// Pyro
 #define ANOMALY_CORE_PYRO /obj/item/assembly/signaler/anomaly/pyro
+/// Hallucination
+#define ANOMALY_CORE_HALLUCINATION /obj/item/assembly/signaler/anomaly/hallucination
```

Comment:
this isn't used anywhere. Are any of the others, even? If not, why not remove this, like we did for status effects?

Reply:
no idea, i added it because it was in the search. i can remove it or leave it for expansions

Reply:
willard, is right you should just remove them all and put their typepaths in place. this is annoying for expansion